instruments/tests/test_thorlabs/test_thorlabs_apt.py

```diff
@@ -79,3 +79,72 @@ def test_apt_hw_info():
             "ThorLabs APT Instrument model ABC-123, "
             "serial 01020304 (HW version 42, FW version a1.a2.a3)"
         )
+
+
+# TESTS FOR APT PIEZO INERTIA ACTUATOR CLASS (APT_PIA) #
+
+
+@pytest.fixture
+def init_kim101():
+    """Return the send, receive value to initialize a KIM101 unit."""
+    stdin = ThorLabsPacket(
+        message_id=ThorLabsCommands.HW_REQ_INFO,
+        param1=0x00, param2=0x00,
+        dest=0x50,
+        source=0x01,
+        data=None
+    ).pack()
+    stdout = ThorLabsPacket(
+        message_id=ThorLabsCommands.HW_GET_INFO,
+        dest=0x01,
+        source=0x50,
+        data=hw_info_data.pack(
+            # Serial number
+            b'\x01\x02\x03\x04',
+            # Model number
+            "KIM101".encode('ascii'),
+            # HW type
+            16,
+            # FW version,
+            0xa1, 0xa2, 0xa3,
+            # Notes
+            "abcdefg".encode('ascii'),
+            # HW version
+            42,
+            # Mod state
+            43,
+            # Number of channels
+            4
+        )
+    ).pack()
```

Comment:
These are just the same parameters as in the existing test, but changed for a KIM101 adapter.
 - Model Number
 - Number of Channels